README.md

```diff
@@ -27,54 +26,52 @@
 **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, and synonyms are provided out-of-the-box.
 For more information about features go to [our documentation](https://docs.meilisearch.com/).
 
-<p align="center">
-  <a href="https://crates.meilisearch.com"><img src="assets/crates-io-demo.gif" alt="crates.io demo gif" /></a>
+<p align="center" id="demo">
+  <img src="assets/movies-web-demo.gif" alt="Web interface gif" />
 </p>
 
-> MeiliSearch helps the Rust community find crates on [crates.meilisearch.com](https://crates.meilisearch.com)
-
-## Features
+## ✨ Features
 * Search as-you-type experience (answers < 50 milliseconds)
 * Full-text search
 * Typo tolerant (understands typos and miss-spelling)
+* Faceted search and filters
 * Supports Kanji characters
 * Supports Synonym
 * Easy to install, deploy, and maintain
 * Whole documents are returned
 * Highly customizable
 * RESTful API
-* Faceted search and filtering
 
-## Get started
+## Getting started
 
 ### Deploy the Server
 
-#### Run it using Digital Ocean
+#### Brew (Mac OS)
 
-[![DigitalOcean Marketplace](assets/do-btn-blue.svg)](https://marketplace.digitalocean.com/apps/meilisearch?action=deploy&refcode=7c67bd97e101)
+```bash
+brew update && brew install meilisearch
+meilisearch
+```
 
-#### Run it using Docker
+#### Docker
 
 ```bash
 docker run -p 7700:7700 -v $(pwd)/data.ms:/data.ms getmeili/meilisearch
 ```
 
-#### Installing with Homebrew
+#### Run on Digital Ocean
 
-```bash
-brew update && brew install meilisearch
-meilisearch
-```
+[![DigitalOcean Marketplace](assets/do-btn-blue.svg)](https://marketplace.digitalocean.com/apps/meilisearch?action=deploy&refcode=7c67bd97e101)
 
-#### Installing with APT
+#### APT (Debian & Ubuntu)
 
 ```bash
 echo "deb [trusted=yes] https://apt.fury.io/meilisearch/ /" > /etc/apt/sources.list.d/fury.list
 apt update && apt install meilisearch-http
 meilisearch
 ```
 
-#### Download the binary
+#### Download the binary (Linux & Mac OS)
```

Comment:
We also support Windows now :)

Reply:
Not from the download-latest script though